CenterForOpenScience/osf.io
# [ENG-6012] User Draft Preprint enpoint

## Purpose

Adds an endpoint that allows for the viewing, cancelling and publishing for draft preprints.


## Changes

- Add view for `/user/<id>/draft_prepints`
- adds DraftPreprintSerializer to ensure type works with FE
- Adds tests


## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify Serializer has type `draft-preprint`, 
- Verify

What are the areas of risk?

Any concerns/considerations/questions that development raised?

## Documentation



## Side Effects



## Ticket

https://openscience.atlassian.net/browse/ENG-6012
